### PR TITLE
fix 876: add support for @tag federation v1 directive

### PR DESCRIPTION
**Pull request by itamar82**
adds support for @tag direction in a federation v1 schema

### CODE REVIEW

File: tests/federation/test_schema.py [hunk 1]
```diff
@@ -18,6 +18,31 @@
 )
 
 
+def test_federation_one_schema_mark_type_tags():
+
```
Comment (rafalp) [inline tests/federation/test_schema.py:22]:
```suggestion
```

**Comment by rafalp** [comment]
Is this all that it took to support `@tag` for federations? Is it only a directive used to put extra metadata on schema elements that has no related logic for supergraph members? I am asking this because I'm not up to date with Apollo's Federation these days. 🤔

**Comment by itamar82** [comment]
> Is this all that it took to support `@tag` for federations? Is it only a directive used to put extra metadata on schema elements that has no related logic for supergraph members? I am asking this because I'm not up to date with Apollo's Federation these days. 🤔

AFAIK it is a directive used within the Apollo Gateway or when the Supergraph is composed so there shouldn't be any implementation necessary in the subgraph, just need to support it.

https://www.apollographql.com/docs/graphos/contracts/#2-fed1-enable-variant-support-for-tag

**Review by rafalp — APPROVED** [review]

**Comment by rafalp** [comment]
Thank you!

**Pull request merged**